File: Learn the Basic/Pattern_printing/Reverse_Letter_Triangle_Pattern.py
def Reverse_Letter_Triangle_Pattern(n):
    for i in range(n):
        for j in range(ord('A'),(ord('A')+(n-i-1))+1):
            print(chr(j),end="")
        print()
n=int(input("Enter the value of n: "))
Reverse_Letter_Triangle_Pattern(n)

File: Learn the Basic/Pattern_printing/Reverse_Letter_Triangle_Pattern.cpp
#include<iostream>
using namespace std;
void Reverse_Letter_Triangle_Pattern(int n){
    for(int i=0;i<n;i++){
        for(char j='A';j<='A'+(n-i-1);j++){
            cout<<j;
        }
        cout<<endl;
    }
}
int main(){
    int n;
    cout<<"Enter the value of n: ";
    cin >> n;
    Reverse_Letter_Triangle_Pattern(n);
    return 0;
}